main.go

```diff
@@ -258,3 +291,22 @@ func main() {
 		glog.Info("Received SIGTERM, exiting gracefully...")
 	}
 }
+
+// Returns intiliazed config, allows local usage (outside cluster) based on provided kubeconfig or in-cluter
+func initKubeConfig(kcLocation string) *rest.Config {
+
+	if kcLocation != "" {
+		kubeConfig, err := clientcmd.BuildConfigFromFlags("", kcLocation)
+		if err != nil {
+			glog.Fatal("unable to build rest config based on provided kubeconfig location. Trying in-cluster configuration")
```

Comment:
This error message contradicts actual behavior. If loading kubeconfig fails, we do `glog.Fatal` here, effectively `os.Exit(255)`ing, see [1]. We should either change the description or change the behavior.

[1] https://godoc.org/github.com/golang/glog#Fatal